docs/docs/responses.mdx

```diff
@@ -167,8 +167,12 @@ Each button in the list of `buttons` should have two keys:
 :::note bypass nlu with buttons
 It's common to use buttons as a shortcut to bypass the machine-learning-based NLU interpreter.
 Messages starting with `/` are sent straight to the
-`RegexInterpreter`, which expects NLU input in a shortened `/intent{entities}` format.
-In the example above, if the user clicks a button, the user input
+`RegexInterpreter`, which expects NLU input in a shortened `/intent{entities}` format, e.g.
+```yaml-rasa {4-8}
+payload: "/inform{{\"planet\": \"saturn\"}}"
```

Comment:
this is what gets rendered for me:
![Bildschirmfoto 2020-12-07 um 11 41 16](https://user-images.githubusercontent.com/1098412/101341294-27869480-3881-11eb-88f2-098de60e1ff6.png)

Do we really need the double `{{`?

Reply:
Yes, again, if you don't write the double `{{`, then it won't work.

Reply:
Has something to do with how regex processes it. I had to find that out the hard way when I built a bot.

Reply:
This is actually why I created this PR in the first place, so people don't have to search on the forum.